e2e/e2e_gov_test.go

```diff
@@ -32,7 +32,7 @@ func (s *IntegrationTestSuite) testWasmStorageStoreExecutorWasm() {
 
 	bytecode, err := os.ReadFile(filepath.Join(localWasmDirPath, executorWasm))
 	if err != nil {
-		panic("failed to read data request Wasm file")
+		panic("failed to read a wasm file")
```

Comment:
Unsure if we should say `oracle program` here as well. As if a user gets an error they might got `wtf is a wasm`.